internal/plan/plan.go

```diff
@@ -65,6 +65,7 @@ type Service struct {
 	Startup     ServiceStartup `yaml:"startup,omitempty"`
 	Override    Override       `yaml:"override,omitempty"`
 	Command     string         `yaml:"command,omitempty"`
+	LogTrim     string         `yaml:"log-trim,omitempty"`
```

Comment:
What time formats should be considered for a default regex?

Reply:
Yeah, I don't know - that would require a bit of research. And on second thoughts, I wonder if the sane default is *not* to do that. It could get messy quite quickly. Maybe we just include a couple of example regexes in the README/docs?

It's also a performance hit, as we've discussed, though it would be nice to know how much of one. Would you be able to make a small Go benchmark of logging via just the `FormatWriter` vs logging via both `FormatWriter` and `TrimWriter` and see what the performance of each is like?

Reply:
Yeah - I think defaults will either quickly get out of hand, or will not match very often despite always causing the performance hit.  Probably better to have some suggestions in docs.

Reply:
Switching back to timestamp parsing - marking this as resolved.